profiler/README.rst

```diff
@@ -0,0 +1,68 @@
+Profiler
```

Comment:
It would be good to update this README to [the latest template](https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst).